samples/authorization/csm_authorization_proxy_server.yaml

```diff
@@ -9,16 +9,16 @@ spec:
     - name: authorization-proxy-server
       # enable: Enable/Disable csm-authorization
       enabled: true
-      configVersion: v1.4.0
+      configVersion: v1.5.0
       forceRemoveModule: true
       components:
       - name: karavi-authorization-proxy-server
         # enable: Enable/Disable csm-authorization proxy server
         enabled: true
-        image: dellemc/csm-authorization-proxy:v1.4.0
-        image: dellemc/csm-authorization-tenant:v1.4.0
-        image: dellemc/csm-authorization-role:v1.4.0
-        image: dellemc/csm-authorization-storage:v1.4.0
+        image: dellemc/csm-authorization-proxy::v1.5.0
```

Comment:
some of the images have an additional semicolon added

Reply:
Fixed.